File: go-sdk/pkg/models/operations/orchestrationgetserverinfo.go
package operations

import (
	"github.com/speakeasy-sdks/formance-sdks/go-client-sdk/pkg/models/shared"
)

type OrchestrationgetServerInfoResponse struct {
	ContentType string
	Error       *shared.Error
	ServerInfo  *shared.ServerInfo
	StatusCode  int64
}

File: ts-sdk/src/sdk/models/operations/orchestrationgetserverinfo.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import * as shared from "../shared";


export class OrchestrationgetServerInfoResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  error?: shared.ErrorT;

  @SpeakeasyMetadata()
  serverInfo?: shared.ServerInfo;

  @SpeakeasyMetadata()
  statusCode: number;
}